src/rules/noPropertyGetterSetterRule.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2019 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as ts from "typescript";
+
+import * as Lint from "../index";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    /* tslint:disable:object-literal-sort-keys */
+    public static metadata: Lint.IRuleMetadata = {
+        ruleName: "no-property-getter-setter",
```

Comment:
From https://github.com/palantir/tslint/issues/3340#issuecomment-494076011:

> But: we should note that any rule to fix this issue should be configured to prefer either way, not hardcoded to just one.

This rule should be named `property-getter-setter`, then, since it's about configuring _how_ to set these up. It should also allow either style preference.

Reply:
> any rule to fix this issue should be configured 

I'm not sure that this rule fixes that issue. We treat this rule as rule about preventing usage of JS properties, not about forcing props style.